HGPlaceholders/Classes/Protocols/NibLoadable.swift

```diff
@@ -16,7 +16,7 @@ import UIKit
  *
  * to be able to instantiate them from the NIB in a type-safe manner
  */
-protocol NibLoadable: class {
+protocol NibLoadable: AnyObject {
```

Comment:
@Idomo  Any reason to use `AnyObject` instead of `class`?

Reply:
As of xCode 13 there is a deprecation warning: `Using 'class' keyword to define a class-constrained protocol is deprecated; use 'AnyObject' instead`